README.md

```diff
@@ -31,25 +31,40 @@ The Javadocs are in the `./javadoc` directory.
 
 ## Relationship to the Iceberg Repository
 
-The `documentation` folder in the [Iceberg repository](https://github.com/apache/iceberg) contains all the markdown docs used by the website.
-The `common` subfolder contains contents used by the landing page.
-The `versioned` subfolder contains the contents different for each version.
+All markdown pages that are part of a versioned released are maintained in the iceberg repository. All pages common across all version
```

Comment:
How about "All markdown pages that are specific to an Iceberg version are maintained..." I think that's slightly more clear.

Reply:
Agreed, updated!